nixos/modules/programs/oblogout.nix

```diff
@@ -0,0 +1,195 @@
+
+with lib;
+
+let cfg = config.programs.oblogout;
+
+in
+{
+  ###### interface
+
+  options = {
+
+    programs.oblogout = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to install OBLogout and create <filename>/etc/oblogout.conf</filename>.
+        '';
+      };
+
+      opacity = mkOption {
+        type = types.int;
+        default = 70;
+        description = ''
+        '';
+      };
+
+      bgcolor = mkOption {
+        type = types.str;
+        default = "black";
+        description = ''
+        '';
+      };
+
+      buttontheme = mkOption {
+        type = types.str;
+        default = "oxygen";
+        description = ''
+        '';
+      };
+
+      buttons = mkOption {
+        type = types.str;
+        default =  "cancel, logout, restart, shutdown, suspend, hibernate, lock";
+        description = ''
+        '';
+      };
+
+      cancel = mkOption {
+        type = types.str;
+        default =  "Escape";
+        description = ''
+        '';
+      };
+
+      shutdown = mkOption {
+        type = types.str;
+        default = "S";
+        description = ''
+        '';
+      };
+
+      restart = mkOption {
+        type = types.str;
+        default = "R";
+        description = ''
+        '';
+      };
+
+      suspend = mkOption {
+        type = types.str;
+        default = "U";
+        description = ''
+        '';
+      };
+
+      logout = mkOption {
+        type = types.str;
+        default = "L";
+        description = ''
+        '';
+      };
+
+      lock = mkOption {
+        type = types.str;
+        default = "K";
+        description = ''
+        '';
+      };
+
+      hibernate = mkOption {
+        type = types.str;
+        default =  "H";
+        description = ''
+        '';
+      };
+
+      cshutdown = mkOption {
+        type = types.str;
+        default = "systemctl poweroff";
+        description = ''
+        '';
+      };
+
+      crestart = mkOption {
+        type = types.str;
+        default = "systemctl reboot";
+        description = ''
+        '';
+      };
+
+      csuspend = mkOption {
+        type = types.str;
+        default = "systemctl suspend";
+        description = ''
+        '';
+      };
+
+      chibernate = mkOption {
+        type = types.str;
+        default = "systemctl hibernate";
+        description = ''
+        '';
+      };
+
+      clogout = mkOption {
+        type = types.str;
+        default = "openbox --exit";
+        description = ''
+        '';
+      };
+
+      cclock = mkOption {
+        type = types.str;
+        default = "xtrlock &";
+        description = ''
+        '';
+      };
+
+      cswitchuser = mkOption {
+        type = types.str;
+        default = "gdm-control --switch-user";
+        description = ''
+        '';
+      };
+
+      csafesuspend = mkOption {
+        type = types.str;
+        default = "safesuspend";
```

Comment:
I could not find `safesuspend`. It is not enabled in none of the `oblogout` packages I found in other distros. So I am going to disable it too.